pkg/commands/pull_request_test.go

```diff
@@ -139,9 +212,33 @@ func TestCreatePullRequest(t *testing.T) {
 				assert.Equal(t, "https://gitlab.com/peter/calculator/merge_requests/new?merge_request[source_branch]=feature/ui", url)
 			},
 		},
+		{
+			testName: "Opens a link to new pull request on gitlab with specific target branch",
+			from: &models.Branch{
+				Name: "feature/commit-ui",
+			},
+			to: &models.Branch{
+				Name: "epic/ui",
+			},
+			remoteUrl: "git@gitlab.com:peter/calculator.git",
+			command: func(cmd string, args ...string) *exec.Cmd {
+				// Handle git remote url call
+				if strings.HasPrefix(cmd, "git") {
+					return secureexec.Command("echo", "git@gitlab.com:peter/calculator.git")
+				}
+
+				assert.Equal(t, cmd, "open")
+				assert.Equal(t, args, []string{"https://gitlab.com/peter/calculator/merge_requests/new?merge_request[source_branch]=feature/commit-ui&merge_request[target_branch]=epic/ui"})
```

Comment:
What happens on those wired nested repo's?  
Like: https://gitlab.com/peter/some_group/calculator  

I can remember working with repos with more than url path 2 parts in gitlab and lots of software braking on them :(

Reply:
If there are any problems with those and they are complex to fix maybe we should then just ignore them as it's more a bug than a feature if you would ask me